crypto/src/lib.rs

```diff
@@ -61,6 +67,7 @@ use self::crypto_lib::sodiumoxide as crypto_impl;
 mod macros;
 
 pub(crate) mod crypto_lib;
+pub(crate) mod utils;
```

Comment:
```suggestion
pub mod utils;
```

Has to be just public. As `crypto` is a crate itself, we won't be able to use `utils` outside, in `exonum`.

Reply:
Or maybe we should reexport these functions putting them into the root scope.